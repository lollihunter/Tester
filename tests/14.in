17 10 20
1 1
5 1
12 7
16 7
6 7
3 10
14 10
9 6
14 5
8 6
13 10
17 2
3 7
15 9
2 10
2 4
12 4
15 7
12 8
17 10
