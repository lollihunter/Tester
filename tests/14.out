10
1 16 13 5 10 8 19 7 14 20 
